lua/lvim/lsp/utils.lua

```diff
@@ -40,7 +40,7 @@ function M.get_client_capabilities(client_id)
   end
 
   local enabled_caps = {}
-  for capability, status in pairs(client.resolved_capabilities) do
+  for capability, status in pairs(client.server_capabilities or client.resolved_capabilities) do
```

Comment:
this gives a annoying warning in newer neovim

maybe sth like this?
```lua
local server_cap = client.server_capaabilities
if vim.version().minor < 7 then
  server_cap = client.resolved_capabilites
end
for capability, status in pairs(server_cap) do
```

what do you think?

Reply:
a warning in the newer version? what does it say? the first part of the condition should've evaluated automatically, so you shouldn't getting any warnings. 

make sure it's not null-ls that's causing the warning, by disabling the setup for it, or maybe fighting with packer a bit lol

Reply:
I found a weird bug that probably messed with `packer.rollback()` so make sure to try this again